app/components/Editor.tsx

```diff
@@ -280,7 +280,16 @@ function Editor(props: Props, ref: React.RefObject<SharedEditor> | null) {
           ref={mergeRefs([ref, handleRefChanged])}
           uploadFile={onUploadFile}
           onShowToast={showToast}
-          embeds={embeds}
+          embeds={embeds.map((e) => {
```

Comment:
Lets memoize

Reply:
Didn't understand! You mean https://lodash.com/docs/4.17.15#memoize?

Reply:
As this one has been converted to functional already can use `useMemo` hook – I want to make sure it isn't mapping to new objects very render

Reply:
Fixed!